src/views/clusteroverview/MonitoringTab/migrations-card/components/MigrationsTable/components/MigrationPolicyTooltip/utils.ts

```diff
@@ -0,0 +1,28 @@
+import { t } from '@kubevirt-utils/hooks/useKubevirtTranslation';
+import { NO_DATA_DASH } from '@kubevirt-utils/resources/vm/utils/constants';
+import { readableSizeUnit } from '@kubevirt-utils/utils/units';
+
+export const getBooleanText = (value: boolean): string => (value ? t('Yes') : t('No'));
+
+export const getBandwidthPerMigrationText = (bandwidth: string | number): string => {
+  if (typeof bandwidth === 'string') return readableSizeUnit(bandwidth);
+  return `${bandwidth}`;
+};
+
+export const getCompletionTimeoutText = (completionTimeout: number): string =>
+  completionTimeout ? `${completionTimeout} sec` : NO_DATA_DASH;
+
+export const migrationsConfigTooltipMap = [
+  {
+    field: 'bandwidthPerMigration',
```

Comment:
those fields values can also be under migrationsDefaultConfigurations and mpObj ?